packages/workbox-cli/test/node/generate-sw-e2e.js

```diff
@@ -17,7 +17,7 @@ describe('Generate SW End-to-End Tests', function() {
   const FILE_EXTENSIONS = ['html', 'css', 'js', 'png'];
 
   before(function() {
-    tmpDirectory = fs.mkdtempSync(path.join(__dirname, 'tmp-'));
+    tmpDirectory = fs.mkdtempSync(path.join(__dirname, 'temp-'));
```

Comment:
It worries me slightly that these tickles are necessary but.. given that it's green now, we can investigate later.